src/common/components/clusterDetail/ProgressBarTexts.tsx

```diff
@@ -25,6 +24,7 @@ export const ProgressBarTexts: React.FC<HostProgressProps> = ({ hosts, hostRole
           filteredHosts.length,
           hostRole === 'master' ? 'control plane node' : 'worker',
         )}`;
+
   const getHostName = (hostRole: HostRole): string => {
```

Comment:
Should be declared outside the component

Reply:
It doesn't even need to be a function :grin: